docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
#1905 is also relevant and worth a read, as it discusses the model(s) of multi-object operations and updates that we are working on.